coalib/settings/ConfigurationGathering.py

```diff
@@ -151,19 +151,17 @@ def load_configuration(arg_list, log_printer, arg_parser=None):
     check_conflicts(cli_sections)
 
     if (
-            bool(cli_sections['default'].get('find_config', 'False')) and
-            str(cli_sections['default'].get('config')) == ''):
-        cli_sections['default'].add_or_create_setting(
-            Setting('config',
-                    re.escape(find_user_config(os.getcwd())),
-                    from_cli=True))
+            bool(cli_sections['cli'].get('find_config', 'False')) and
+            str(cli_sections['cli'].get('config')) == ''):
+        cli_sections['cli'].add_or_create_setting(
+            Setting('config', re.escape(find_user_config(os.getcwd()))))
```

Comment:
so why do you remove it again? We still want the correct deprecation warning don't we?